doc/Changelog.md

```diff
@@ -1,8 +1,10 @@
 Changelog {#Changelog}
 =========
 
-# git master
+# Release 1.7.0 (24-Mar-2016)
 
+* [51](https://github.com/BlueBrain/Brion/pull/51):
+  Empty target does not rise an exeption anymore
```

Comment:
Again, not helpful (as in the commit msg already). Think of what you want to know reading this, not what you did writing the code. Should be something 'Empty target does foo instead of raising an exception'

@favreau as the original committer.